lib/Dialect/AIEX/Transforms/AIECtrlPacketToDma.cpp

```diff
@@ -125,7 +125,7 @@ struct AIECtrlPacketToDmaPass : AIECtrlPacketToDmaBase<AIECtrlPacketToDmaPass> {
 
               StringRef metadata = builder.getStringAttr(shimDmaAllocName);
               builder.create<NpuDmaMemcpyNdOp>(
-                  builder.getUnknownLoc(), 0, 0, newBlockArg,
+                  builder.getUnknownLoc(), newBlockArg, SmallVector<Value>{},
```

Comment:
There seems to be one extra redundant `SmallVector<Value>{}` argument passed to the builder.

Reply:
Nice catch! Thank you Erwei! :)